backend/src/Squidex.Domain.Apps.Core.Model/Contents/Json/JsonWorkflowTransition.cs

```diff
@@ -1,12 +1,10 @@
-// ==========================================================================
```

Comment:
What are the changes here?

Reply:
I'm unable to spot anything, looks like a special character.